exporters/trace/stackdriver/src/test/java/io/opencensus/exporter/trace/stackdriver/StackdriverV2ExporterHandlerProtoTest.java

```diff
@@ -130,38 +129,24 @@
       TimedEvents.create(networkEventsList, DROPPED_NETWORKEVENTS_COUNT);
   private static final SpanData.Links links = SpanData.Links.create(linksList, DROPPED_LINKS_COUNT);
   private static final Map<String, AttributeValue> EMPTY_RESOURCE_LABELS = Collections.emptyMap();
-  private static final Resource AWS_EC2_INSTANCE_RESOURCE =
-      AwsEc2InstanceResource.create("my-project", "us-east-1", "my-instance");
-  private static final Resource GCP_GCE_INSTANCE_RESOURCE =
-      GcpGceInstanceResource.create("my-project", "us-east1", "my-instance");
-  private static final Resource K8S_CONTAINER_RESOURCE =
-      K8sContainerResource.create("cluster", "namespace", "pod", "container");
-  private static final ImmutableMap<String, AttributeValue> EC2_RESOURCE_LABELS =
-      ImmutableMap.of(
-          createResourceLabelKey(AwsEc2InstanceResource.ACCOUNT_ID_KEY),
-          toStringAttributeValueProto("my-project"),
-          createResourceLabelKey(AwsEc2InstanceResource.REGION_KEY),
-          toStringAttributeValueProto("us-east-1"),
-          createResourceLabelKey(AwsEc2InstanceResource.INSTANCE_ID_KEY),
-          toStringAttributeValueProto("my-instance"));
-  private static final ImmutableMap<String, AttributeValue> GCE_RESOURCE_LABELS =
+  private static final Resource CUSTOM_RESOURCE =
```

Comment:
Just to double check: for Stackdriver Trace exporter we don't need to do resource label mappings? (use the modified labels as-is?)